query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -17,13 +17,6 @@
 public class InfinispanQueryStatisticsInfo implements InfinispanQueryStatisticsInfoMBean {
 
```

Comment:
getSearchQueryExecutionAvgTime should also be just delegated instead of being computed.